### PR TITLE
47 update directors page

### PR DESCRIPTION
**Pull request by TroyNWest**
Updates the directors information page to reflect current staffing changes.

### CODE REVIEW

File: pug/src/staff/directors.pug [hunk 1]
```diff
@@ -50,16 +53,13 @@ block content
                 div(class='card h-100')
                     div(class='card-header') Vice President, Bewhiskered Director, Director of Event Administration
```
Comment (FaelanGalanti) [inline pug/src/staff/directors.pug:54]: Can you move the Bewhiskred Director role and Bewhiskered footer to me? I'm keeping the role for now.

File: pug/src/staff/directors.pug [hunk 2]
```diff
@@ -131,6 +125,28 @@ block content
...
+            div(class='col mb-4')
+                div(class='card h-100')
+                    div(class='card-header') Director of Organization Operations
+                    div(class='card-body')
+                        h1(class='card-title mb-4') Pat
+                        img(class='icon float-left' src='/img/director-icons/pat.png' alt='Pat')
+                        p(class='card-text')
+                        | Hello! My name is Pat (I sometimes go by my sona Axel); I'm a civil engineer by day but by night I run the boring but important Org Ops division, where we've cut a deal with the devil the departments all do nothing at conventions but everything to make the organization possible, in the form of HR (Yes, we know we blew calling it Furry Resources), IT, Purchasing, and Accounting!
+                    div(class='card-footer')
+                        | <Division description>
```
Comment (FaelanGalanti) [inline pug/src/staff/directors.pug:137]: Nit: missing description.

File: pug/src/staff/directors.pug [hunk 3]
```diff
@@ -131,6 +125,28 @@ block content
...
+            div(class='col mb-4')
+                div(class='card h-100')
+                    div(class='card-header') Director of Guest Relations
+                    div(class='card-body')
+                        h1(class='card-title mb-4') Southbridge
+                        img(class='icon float-left' src='/img/director-icons/southbridge.png' alt='Southbridge')
+                        p(class='card-text')
+                            | Hey! I'm Southbridge! I've been with the fandom for quite a while and with NCAS in one way or another since the beginning! I love our community and want to do everything I can to help support it. In my boring human life I'm a cybersecurity analyst and full stack developer. Looking forward to put my skills into creating the best events we can!
+                    div(class='card-footer')
+                        | <Division description>
```
Comment (FaelanGalanti) [inline pug/src/staff/directors.pug:148]: Nit: missing description.

Comment (TroyNWest) [reply pug/src/staff/directors.pug:148]: Do we have a description?

File: pug/src/staff/directors.pug (hunk 1 above)
Comment (TroyNWest) [reply pug/src/staff/directors.pug:54]: Will move those over in the next push.

File: pug/src/staff/directors.pug (hunk 2 above)
Comment (TroyNWest) [reply pug/src/staff/directors.pug:137]: Do we have a description?

**Review by FaelanGalanti — APPROVED** [review]

**Pull request merged**